### PR TITLE
feat: integrate Symbol editor

### PR DESCRIPTION
**Pull request by lorentzfb**
This PR is an example of basic integration/use of the symbol editor.

NOTE: Only "edit" symbol will load the symbol into the editor, other actions (new and update) is not implemented.

See this example of how to integrate further: https://github.com/equinor/dugtrio-experimental/blob/main/users/loba/symbol-editor/src/App.tsx

### CODE REVIEW

File: web/pages/edit/index.page.tsx [hunk 1]
```diff
@@ -390,6 +407,21 @@ const Edit: NextPage<EditPageProps> = ({ theme }) => {
 		},
 	];
 
+	const onEditorEvent = (event: SymbolEditorEvent) => {
+		console.log(`EDITOR-EVENT:${event.type}:${event.reason}`);
+		console.log('Event data: ', event.data);
+		console.log('Symbol state: ', event.symbolState);
+	};
+
+	const editorStyle: React.CSSProperties = {
```
Comment (DzianisMakeichyk) [inline web/pages/edit/index.page.tsx:416]: To styles?

File: web/pages/edit/index.page.tsx [hunk 2]
```diff
@@ -42,6 +42,9 @@ import {
 	SymbolUploadStore,
 } from '../../store';
 import React from 'react';
+import EngineeringSymbolEditor from '../../components/symbolEditor/EngineeringSymbolEditor';
+import { SymbolEditorEvent } from '../../components/symbolEditor/models/EditorEvent';
+import { EditorCommandMessage } from '../../components/symbolEditor/models/EditorCommand';
```
Comment (DzianisMakeichyk) [inline web/pages/edit/index.page.tsx:47]: Mb add some `index.ts` to make import more beautiful ;)

Comment (lorentzfb) [reply web/pages/edit/index.page.tsx]: Done

**Pull request merged**